docs/source/augly.rst

```diff
@@ -0,0 +1,23 @@
+augly package
+=============
+
+Subpackages
+-----------
+
+.. toctree::
+   :maxdepth: 4
+
+   augly.audio
+   augly.image
+   augly.tests
+   augly.text
+   augly.utils
```

Comment:
im borderline about `augly.utils` -- i think it's nice to show off what assets we have, but maybe we don't need to publicize every file in there